pallets/loans/src/entities/loans.rs

```diff
@@ -489,3 +491,47 @@ impl<T: Config> ActiveLoan<T> {
 		self.schedule.maturity = crate::types::Maturity::fixed(duration);
 	}
 }
+
+/// Data containing an active loan with extra computed.
+#[derive(Encode, Decode, Clone, PartialEq, Eq, RuntimeDebugNoBound, TypeInfo, MaxEncodedLen)]
+#[scale_info(skip_type_params(T))]
+#[cfg_attr(feature = "std", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(feature = "std", serde(bound = ""))]
+pub struct ActiveLoanInfo<T: Config> {
```

Comment:
This is a bit tricky and probably useful for futures runtime APIs.
The line with `bound = ""` is mandatory for cases where you have both, the `scale_info` and `derive(serde)` lines. Serde defaults to adding `Serialize` and `Deserialize` bounds to `T`,  but T, in this case, does not need to be serializable, it's only used to fetch information from associated types. With `bounds = ""`, we remove those implicit bounds from serde.

Reply:
Thanks for clarifying this. So this is the `serde` equivalent of `scale_info(skip_type_params(T)` more or less?

Reply:
Yeah! More or less. Also, if there were other bounds, and you do not want to "reset them" you should add `bound = "K: MyOtherBound"`. That way you've reset the `Serialize` bound to `T` and `K` but not the `MyOtherBound` to `K`.